.github/workflows/deno.yml

```
@@ -0,0 +1,33 @@
+name: Deno
+
+on:
+    push:
+        branches: [main]
+    pull_request:
+        branches: [main]
+
+jobs:
+    fmt-lint:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+
+            - uses: denoland/setup-deno@main
```

Why not v1 like they specify in their README?
https://github.com/denoland/setup-deno

```suggestion
            - uses: denoland/setup-deno@v1
```

---

I just copied, didn't know about this.